src/test/java/com/tsunderebug/speedrun4j/user/UserTest.java

```diff
@@ -0,0 +1,141 @@
+package com.tsunderebug.speedrun4j.user;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserTest {
+
+    static User testUser;
+
+    @BeforeAll
+    static void createTestUserFromId() {
+        testUser = User.fromID("lundylizard");
```

Comment:
Tests should ideally not access the internet. Perhaps a way to parse a user from a JSON string or inputstream, or setting up a mock API server that conforms to SRC's are valid options.